.github/workflows/contracts.yaml

```diff
@@ -8,6 +8,23 @@ on:
   workflow_dispatch:
 
 jobs:
```

Comment:
@michalinacienciala Could you please review the change here?

Reply:
Looking good!